### PR TITLE
Updated the task cost to estimated_computation_cost

### PR DESCRIPTION
**Pull request by Pbarbosa92**
This PR fixes a breaking change from https://github.com/inductiva/inductiva-web-api/pull/1270

Only merge after https://github.com/inductiva/inductiva-web-api/pull/1270 gets merged

### CODE REVIEW

**Review by HenriquePreto — APPROVED** [review]

**Review by hpenedones — CHANGES_REQUESTED** [review]
Please change to estimated_ rather calculated_ ... Thanks!

File: inductiva/tasks/task.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def __init__(self, **kwargs):
         self.computation_start_time = None
         self.computation_end_time = None
         self.end_time = None
-        self.cost = None
+        self.estimated_cost_per_hour = None
```
Comment (bgregoinductiva) [inline inductiva/tasks/task.py:99]: The JSON response parameter is actually `estimated_computation_cost`.

Comment (Pbarbosa92) [reply inductiva/tasks/task.py]: Fixed it

**Review by bgregoinductiva — APPROVED** [review]

File: inductiva/tasks/task.py [hunk 2]
```diff
@@ -239,8 +239,9 @@ def __str__(self):
         table_str += f"\n{wall_time_table}"
         table_str += f"\nTime breakdown:\n{time_metrics_table}"
         table_str += f"\nData:\n{data_metrics_table}\n"
-        if self.cost:
-            table_str += f"\nCost ($US): {self.cost}\n"
+        if self.estimated_computation_cost:
+            table_str += ("\nEstimated computation cost ($US): "
```
Comment (sergiomsantos) [inline inductiva/tasks/task.py:243]: It should be `US$` and not `$US`:

```suggestion
            table_str += ("\nEstimated computation cost (US$): "
```

**Pull request merged**